subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/ArtifactsTestListener.kt

```diff
@@ -73,20 +75,22 @@ internal class ArtifactsTestListener(
                         )
                     )
                 }
-                TestResult.Complete(artifacts)
+                if (artifacts.isSuccess()) {
+                    TestResult.Complete(artifacts)
```

Comment:
What about unwrapping from `Result` to actual value

Reply:
Done! Thanks for pointing it out.
Unwrapping also leads to fixing one possible Lost cause